src/handlerCode.js

```diff
@@ -1,7 +1,17 @@
+let handler, handlerError;
+try {
+  handler = require('../RELATIVE_PATH');
```

Comment:
I can't remember understanding this before -- what's this `../RELATIVE_PATH` thing? Does it get replaced? a comment could be useful here. Unrelated to this change (the movement makes sense from the PR reasoning, although a comment about why it's here could be handy for source readers)

Reply:
Yes, this "code" actually gets imported as text with the capitalized pieces replaced. This is nice because we get syntax highlighting / formatting on this file but it doesn't have to be real / work.

Reply:
It's out of the scope of this work, but that would be a really good comment if you don't mind bringing it in here (it's a little small to warrant an issue)